.chloggen/debug-exporter-normal-verbosity-traces.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

```suggestion
change_type: enhancement
```

Based on https://github.com/open-telemetry/opentelemetry-collector/blob/486b0645aac1eb03407139ae7840a9523f6ae8a2/exporter/debugexporter/README.md?plain=1#L125, this is not a breaking change. I am not comfortable setting the expectation that people should rely on the output being stable. If people want a machine readable output, they should use the file exporter

---

It actually says in the [README](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.102.0/exporter/debugexporter/README.md#warnings) that changes in output format are not considered breaking. :sweat_smile: